vkcs/firewall/resource_secgroup.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/gophercloud/gophercloud/openstack/networking/v2/extensions/security/groups"
 	igroups "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/firewall/v2/groups"
 	iattributestags "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/networking/v2/attributestags"
+
+	useconstants "github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/services/networking"
```

Comment:
It's better to name it according to the same pattern as above: `inetworking`. Also, there is no need in empty line above it.